applications/StructuralMechanicsApplication/custom_conditions/line_load_condition_2d.cpp

```diff
@@ -103,217 +103,195 @@ void LineLoadCondition2D::CalculateAll(
-        if ( CalculateResidualVectorFlag == true )
-        {
-            if ( gauss_pressure != 0.0 )
-            {
-                CalculateAndAddPressureForce( rRightHandSideVector, row( Ncontainer, point_number ), normal, gauss_pressure, integration_weight );
+        if ( CalculateResidualVectorFlag ) {
+            if ( gauss_pressure != 0.0 ) {
+                CalculateAndAddPressureForce( rRightHandSideVector, row( rNcontainer, point_number ), normal, gauss_pressure, integration_weight );
             }
         }
 
         array_1d<double,3> gauss_load = line_load;
-        for (unsigned int ii = 0; ii < number_of_nodes; ++ii)
-        {
-            if( GetGeometry()[ii].SolutionStepsDataHas( LINE_LOAD ) )
-            {
-                noalias(gauss_load) += ( Ncontainer( point_number, ii )) * GetGeometry()[ii].FastGetSolutionStepValue( LINE_LOAD );
+        for (IndexType ii = 0; ii < number_of_nodes; ++ii) {
+            if( r_geometry[ii].SolutionStepsDataHas( LINE_LOAD ) ) {
+                noalias(gauss_load) += ( rNcontainer( point_number, ii )) * r_geometry[ii].FastGetSolutionStepValue( LINE_LOAD );
             }
         }
-        for (unsigned int ii = 0; ii < number_of_nodes; ++ii)
-        {
-            unsigned int base = ii * block_size;
 
-            for(unsigned int k = 0; k < dimension; ++k)
-            {
-                rRightHandSideVector[base + k] += integration_weight * Ncontainer( point_number, ii ) * gauss_load[k];
+        for (IndexType ii = 0; ii < number_of_nodes; ++ii) {
+            const IndexType base = ii * block_size;
+
+            for (IndexType k = 0; k < dimension; ++k) {
+                rRightHandSideVector[base + k] += integration_weight * rNcontainer( point_number, ii ) * gauss_load[k];
             }
         }
     }
 
-    //if (this->HasRotDof()) this->CalculateAndAddWorkEquivalentNodalForcesLineLoad(gauss_load,rRightHandSideVector);
-
     KRATOS_CATCH( "" )
 }
 
-//***********************************************************************
-//***********************************************************************
+/***********************************************************************************/
+/***********************************************************************************/
 
 void LineLoadCondition2D::CalculateAndSubKp(
-    Matrix& K,
-    const Matrix& DN_De,
-    const Vector& N,
+    Matrix& rK,
+    const Matrix& rDN_De,
+    const Vector& rN,
     const double Pressure,
     const double IntegrationWeight
     ) const
 {
     KRATOS_TRY
 
     Matrix Kij( 2, 2 );
-    Matrix Cross_gn( 2, 2 );
 
-    //TODO: decide what to do with thickness
-    //const double h0 = GetProperties()[THICKNESS];
-    const double h0 = 1.00;
-    Cross_gn( 0, 0 ) = 0.0;
-    Cross_gn( 0, 1 ) = -h0;
+    const auto& r_properties = GetProperties();
+    const double h0 = r_properties.Has(THICKNESS) ? r_properties.GetValue(THICKNESS): 1.0;
```

Comment:
this is a behavior change
How does THICKNESS make sense for a line?

Reply:
This is discussion it was in a TODO. What has thickness is in fact the solid

> I said it manymanymanymanymanymanymany times already, due to the cosmetic changes this is not that easy to review any more, I don't have the time to do it today.
> 
> Can you at least put the cosmetic in a different commit in the future?

Reply:
shouldn't it then only be done in the 2D case?

Reply:
ping

Reply:
LineCondition2D3 and LineCondition2D2 are by definition 2D

Reply:
"line_load_condition_2d.cpp"

Reply:
the name is wrong, this can also be used in 3d (see other parts in the code)

Reply:
we should rename it btw

Reply:
Ok, we can rename it, but not now. this is used in other PR, i will fix only the thickness correction

Reply:
sure, thx

Reply:
Fixed